resources/views/designer/designer.blade.php

```diff
@@ -1,6 +1,8 @@
 @extends('layouts.layout')
 @section('content')
-    <div id="designer-container"></div>
```

Comment:
Undefined variable: process (View: /home/circleci/builds/feature-369/resources/views/designer/designer.blade.php)

Reply:
Now the process is an optional parameter in the controller